lib/nx_signal/transforms.ex

```diff
@@ -0,0 +1,23 @@
+defmodule NxSignal.Transforms do
+  import Nx.Defn
+
+  deftransform fft_nd(tensor, opts \\ []) do
+    Enum.reduce(0..(Nx.rank(tensor) - 1), tensor, fn el, acc ->
+      if el in Keyword.get(opts, :ignore, []) do
```

Comment:
I think you should receive the :axes option, and default it to [-1].

Then, you just reduce on :axes